sources/packages/web/src/services/http/common/HttpClient.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+const apiUrl = process.env.NODE_ENV === 'production' ?
+    "/api" : "http://localhost:3000/api";
```

Comment:
localhost??

Reply:
If we are running the application outside our local env, the "/api" relative address will be used, otherwise, we will be using the localhost for our local development, right?